README.md

```diff
@@ -1 +1,18 @@
-# das-apprentice-learning
\ No newline at end of file
+
+DAS Capture The Flag
+
+# das-apprentice-learning
+Capture the Flag Game is a .NET Core 3.0 web application.  It is a multi-player browser game.
+Once two user's have connected and selected start the game will begin.
+
+The repository can be found here: https://github.com/SkillsFundingAgency/das-apprentice-learning
+
+# Database
+
+# Tests
+
+# Authors
+- Ben Stone
+- Marie Dale
+- Dan Beavon
```

Comment:
These could include links to the respective github profiles

Reply:
Tried again, some sections left blank for now but the template is fully.  I can update the contributing section once created